### ISSUE TITLE
Chrome GPU process not sandboxed

### ISSUE THREAD

**Issue by spsch** [haskellcamargo/skype-unofficial-client#48]
At first, well done, thanks for this port of Skype client, nice piece of work. Here is the issue:
If client is started from terminal emulator as common user, client will open, but:

`[0519/102516:INFO:nw_package.cc(175)] /opt/skype_unofficial_client/skype
[7355:7355:0519/102516:ERROR:sandbox_linux.cc(340)] InitializeSandbox() called with multiple threads in process gpu-process
[7309:7309:0519/102517:ERROR:browser_main_loop.cc(253)] GLib-GObject: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
[7309:7309:0519/102517:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[7309:7309:0519/102517:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[7309:7309:0519/102517:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[7309:7309:0519/102517:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[7309:7309:0519/102517:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
NOT SANDBOXED
`

This should be connected to [Chromium issue](https://bugs.chromium.org/p/chromium/issues/detail?id=264818) reported on official site.

Here is the configuration:

`Chrome version Chrome/50.0.2661.102`
`Operating system   Linux 4.2.0-36-generic`
`Software rendering list version    10.18`
`Driver bug list version    8.59`
`ANGLE commit id    c46018b8598d`
`2D graphics backend    Skia`
`Command Line Args  --window-depth=24 --flag-switches-begin --flag-switches-end`
`Initialization time    85`
`In-process GPU false`
`Sandboxed  false`
`GPU0   VENDOR = 0x1002, DEVICE= 0x9553`
`Optimus    false`
`AMD switchable false`
`Driver vendor  Mesa`
`Driver version 11.0.2`
`GL_VENDOR  X.Org`
`GL_RENDERER    Gallium 0.4 on AMD RV710 (DRM 2.43.0, LLVM 3.6.2)`
`GL_VERSION 3.0 Mesa 11.0.2`
`XDG_CURRENT_DESKTOP    XFCE`
`GDMSESSION xubuntu`
`Compositing manager    Yes`
`Direct rendering   Yes`
`Reset notification strategy    0x8252`
`GPU process crash count    0`

**Comment by spsch** [comment]
Also, when terminated by SIGINT (signal(2)) / Ctrl+C client is closed, but some request context is missing for CSC servers:

`^C[9075:9109:0519/104959:ERROR:nss_ocsp.cc(613)] No URLRequestContext for NSS HTTP handler. host: s2.symcb.com`
`[9075:9109:0519/104959:ERROR:nss_ocsp.cc(613)] No URLRequestContext for NSS HTTP handler. host: s1.symcb.com`
`[9075:9109:0519/104959:ERROR:nss_ocsp.cc(613)] No URLRequestContext for NSS HTTP handler. host: s2.symcb.com`
`[9075:9109:0519/104959:ERROR:nss_ocsp.cc(613)] No URLRequestContext for NSS HTTP handler. host: s2.symcb.com`
`[9075:9109:0519/104959:ERROR:nss_ocsp.cc(613)] No URLRequestContext for NSS HTTP handler. host: s1.symcb.com`
`^C`

[Similar](https://bitbucket.org/chromiumembedded/cef/issues/1490/linux-no-urlrequestcontext-for-nss-http) has been resolved in embedded chrome.

**Comment by lovesegfault** [comment]
I'm having similar issues

```
meurer@mainframe ~> skype-desktop 
[0606/144457:INFO:nw_package.cc(175)] ./skype --enable-spell-checking
[13227:13227:0606/144458:ERROR:browser_main_loop.cc(253)] GLib-GObject: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
[13227:13227:0606/144458:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[13227:13227:0606/144458:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[13227:13227:0606/144458:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[13227:13227:0606/144458:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[13227:13227:0606/144458:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
```

**Comment by esordonez** [comment]
bemeurer can yo solve this problem? I have the same ouput:

λ esor ~ → /opt/skype_unofficial_client/skype
[0617/225329:INFO:nw_package.cc(175)] /opt/skype_unofficial_client/skype
[11327:11327:0617/225329:ERROR:browser_main_loop.cc(253)] GLib-GObject: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
[11327:11327:0617/225329:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[11327:11327:0617/225329:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[11327:11327:0617/225329:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[11327:11327:0617/225329:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[11327:11327:0617/225329:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[11327:11327:0617/225329:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed

**Comment by lovesegfault** [comment]
@eriksantiago91 Nope. No solution so far, still waiting for anything from the maintainer

**Comment by esordonez** [comment]
@bemeurer i decided to install another alternative from aur
